### DISCUSSION TITLE
Improphe also employed on 60m Sentinel bands?

### DISCUSSION THREAD

**Discussion by jo-spek** [davidfrantz/force#266] [Q&A] [answered]
Hi,

since it was not specifically mentioned anywhere, I would like to know whether the 60m bands of Sentinel-2 are also enhanced with the Improphe code. I set all my output images of L2PS to be at 20m resolution and in my products, the bands do all indeed have 20m resolution (also the 60m bands have 20m pixels with different values, i.e. not a bunch of 20m pixels with the same value but actual distinct values). In the parameter file I set the resampling method to cubic convolution and the resolution merging to Improphe. So, if I'm getting it right, my 10m bands are resampled with cubic convolution to 20m. But what about my 60m bands? Are they improphed? Or is something else happening here?

Thanks a lot for your help!

**Comment by davidfrantz** [comment, marked as answer]
Hi @jo-spek,

please have a look here: https://force-eo.readthedocs.io/en/latest/components/lower-level/level2/format.html#product-type

FORCE does not output any of the 60m bands and only uses them internally for cloud detection or atmospheric correction. 

Best,
David

**Reply by jo-spek** [reply]
Uff, of course. Sometimes one gets hung up on the tiniest brainfarts. Thanks a lot David for your kind replies even to the smallest problems!

If anybody happens to end up in this thread: Sentinel-2 collects 13 bands. 3 bands are at 60m resolution but they are not commonly used for spectral analyses. The bands commonly used for analyses are the 10 other bands, 4 of which are at 10m resolution and 6 of which are at 20m resolution.

**Reply by davidfrantz** [reply]
no worries ;)
